examples/use-existing-iam-roles-multi-region/README.md

```diff
@@ -0,0 +1,68 @@
+# Use Existing IAM Roles Example
+
+```hcl
+
+provider "lacework" {}
+
+provider "aws" {
+  region = "us-west-1"
+}
+
+provider "aws" {
+  alias  = "usw2"
+  region = "us-west-2"
+}
+
+// Create global resouces, includes lacework cloud integration
+// Use existing cross account IAM role
+module "lacework_aws_agentless_scanning_global" {
+  source = "../.."
+
+  global                    = true
+
+  use_existing_cross_account_role = true
+  use_existing_event_role         = true
+  use_existing_execution_role     = true
+  use_existing_task_role          = true
+
+  cross_account_role_arn                = "arn:aws:iam::123456789012:role/my-lacework-ca-role"
+  external_id                           = "AbC123dEf456"
+  agentless_scan_ecs_task_role_arn      = "arn:aws:iam::123456789012:role/my-lacework-ecs-task-role"
+  agentless_scan_ecs_execution_role_arn = "arn:aws:iam::123456789012:role/my-lacework-ecs-execution-role"
+  agentless_scan_ecs_event_role_arn     = "arn:aws:iam::123456789012:role/my-lacework-ecs-event-role"
+
+  // Suffix can be used to "predict" what resouce ARNs will be when manually creating IAM policies ahead of time
+  suffix = "abc123"
```

Comment:
In this case is the "real" ARN the supplied ARN with the suffix added? for example:

```
arn:aws:iam::123456789012:role/my-lacework-ca-role-abc123
```

Reply:
In this example, we're using existing IAM roles. so the suffix will only be applied to resources we're creating with TF, so that would be the bucket, secret, ecs cluster, etc. but not any of the IAM roles. So the ARN of the existing role would be whatever you make it to be regardless of what you put in for suffix.

Reply:
I actually have a guide on how to create the policies and roles manually that might be worth publishing to our documentation that ties to this functionality. This way customers know how to manually create the policies and roles and then properly supply them to the TF code.

Reply:
I see, this makes sense.

If we want this example to work with organization-type deployments then we need to do more work if the snapshot-role resource is going to be provided manually. In this case we will need to set the role name (the name, not the ARN) on the task definition.